priv/apps/itest/lib/poller.ex

```diff
@@ -227,6 +237,57 @@ defmodule Itest.Poller do
     {:ok, tx} = Ethereumex.HttpClient.eth_get_transaction_by_hash(response["transactionHash"])
 
     {:ok, reason} = Ethereumex.HttpClient.eth_call(Map.put(tx, "data", tx["input"]), tx["blockNumber"])
-    _ = Logger.info("Revert reason for #{inspect(response)}: #{inspect(Itest.Transactions.Encoding.to_binary(reason))}")
+    hash = response["transactionHash"]
+
+    _ =
+      Logger.info(
+        "Revert reason for #{inspect(hash)}: revert string: #{inspect(reason)} revert string: #{
```

Comment:
Didn't we have a module function that parsed out the reasoning?

Reply:
yes and no, this one still doesn't work well. IDK why.